libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -420,14 +430,18 @@ export class CreatorNode {
   }
 
   async uploadTrackAudioV2(file: File, onProgress: ProgressCB) {
-    return this.uploadFileV2(file, onProgress, 'audio')
+    return await this.uploadFileV2(file, onProgress, 'audio')
```

Comment:
fyi @theoilie

Reply:
does this have to do with how it's used in a generator function or something? otherwise afaik `return await` is redundant here since it's still returning a promise that has to be awaited elsewhere either way

Reply:
so eslint says it's best to await any promise in an async function to improve error stack. if the promise fails itll say "inside uploadTrackAudioV2" function instead of the caller, and of course if that caller doesn't await it could bubble away multiple levels, missing a helpful stacktrace

Reply:
@theoilie lmk if you dont have eslint enabled on save in your editor, also to run lint before commit you can also do `npm run lint:fix`

Reply:
ah gotcha, that sounds like a better reason than the "no-return-await" eslint rule 👍 

I disabled formatting on save because it does weird things in the audius-protocol repo across sub projects, but I'll make sure to run npm run lint:fix next time, thanks